lib/ruby_lsp/executor.rb

```diff
@@ -229,6 +229,8 @@ def did_change_watched_files(changes)
         end
       end
 
+      # TODO: document once we are happy with
```

Comment:
Let's create an issue instead of leaving a todo comment.

Reply:
https://github.com/Shopify/ruby-lsp/issues/1465

(will remove todo)